learn/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM cockpit/infra-base
+LABEL maintainer='cockpit-devel@lists.fedorahosted.org'
+
+RUN dnf -y update \
+ && dnf -y install \
+        lighttpd \
+        python3 \
+        python3-scikit-learn \
+ && dnf clean all
+
+RUN mkdir -p /cache/data \
+ && chown -R user /home/user /cache/data \
+ && ln -snf /learn/predict-tests /cache/predict
+
+COPY * /learn/
+WORKDIR /cache
+ENV TEST_DATA=/cache
+ENV PATH=$PATH:/learn
+
+EXPOSE 8080
+STOPSIGNAL SIGQUIT
+CMD [ "/bin/bash" ]
+
+# To run lighttpd container
+# CMD ["lighttpd", "-D", "-f", "/learn/lighttpd.conf"]
```

Comment:
The documentation says how to use this as a POD with a running web server. So maybe actually use the lighttpd as standard command, and  leave it to `make learn-shell` to debug?

Reply:
Ack, understood. Thanks!